tests/testthat/test_incometax_single.R

```diff
@@ -2,9 +2,6 @@
 context("Individual income tax")
 
 test_that("income tax checks", {
-  # no fy.year
-  expect_error(income_tax(1), regexp = "fy.year is missing")
-  
```

Comment:
Something like:
`#no fy.year`
` expect_equal(income_tax(30e3), 2397)`

Reply:
Actually that would fail if the year changes. Something like this I think is better:
`expect_equal(income_tax(30e3), income_tax(30e3,fy.year(format(as.Date(Sys.Date()),"%Y"))))`